### PR TITLE
Boilerplate update

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Updated the boilerplate

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: boilerplate/lyft/golang_support_tools/go.mod [hunk 1]
```diff
@@ -1,14 +1,12 @@
-module github.com/flyteorg/boilerplate
+module github.com/lyft/boilerplate
```
Comment (wild-endeavor) [inline boilerplate/lyft/golang_support_tools/go.mod:1]: why are we reverting these?

File: boilerplate/lyft/golang_test_targets/download_tooling.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ set -e
 # In the format of "<cli>:<package>" or ":<package>" if no cli
 tools=(
   "github.com/vektra/mockery/cmd/mockery"
-  "github.com/flyteorg/flytestdlib/cli/pflags"
+  "github.com/lyft/flytestdlib/cli/pflags"
```
Comment (wild-endeavor) [inline boilerplate/lyft/golang_test_targets/download_tooling.sh:20]: here too, and a few other places.

File: pull_request_template.md [hunk 3]
```diff
@@ -1,8 +1,3 @@
-## Read then delete
```
Comment (wild-endeavor) [inline pull_request_template.md:1]: is this no longer true?

**Review by wild-endeavor — COMMENTED** [review]
There seems to be a lot going on here, I know there were some shellcheck nits that went in, but do you know about all the other stuff?

Thanks.

File: Makefile [hunk 4]
```diff
@@ -12,6 +12,7 @@ LD_FLAGS="-s -w -X $(PACKAGE)/version.Version=$(GIT_VERSION) -X $(PACKAGE)/versi
 
 .PHONY: update_boilerplate
 update_boilerplate:
+	@curl https://raw.githubusercontent.com/flyteorg/boilerplate/master/boilerplate/update.sh -o boilerplate/update.sh
```
Comment (evalsocket) [inline Makefile:15]: @EngHabu I am handling `update.sh` script in  makefile rather then shell script itself

File: boilerplate/flyte/end2end/Makefile [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# WARNING: THIS FILE IS MANAGED IN THE 'BOILERPLATE' REPO AND COPIED TO OTHER REPOSITORIES.
+# ONLY EDIT THIS FILE FROM WITHIN THE 'FLYTEORG/BOILERPLATE' REPOSITORY:
+#
+# TO OPT OUT OF UPDATES, SEE https://github.com/flyteorg/boilerplate/blob/master/Readme.rst
+
+.PHONY: end2end_execute
+end2end_execute:
+	chmod a+x ./boilerplate/flyte/end2end/end2end.sh
```
Comment (evalsocket) [inline boilerplate/flyte/end2end/Makefile:8]: I added `chmod` for solving the permission issue but I am not sure why it started after migration

**Comment by evalsocket** [comment]
@EngHabu @wild-endeavor Pr is ready for review

File: boilerplate/flyte/end2end/Makefile (hunk 5 above)
Comment (EngHabu) [reply boilerplate/flyte/end2end/Makefile:8]: You can do this locally... and then add the file to git... I believe this is one of the things that get persisted in git... does that make sense?

``
chmod a+x ....
git add -A
git commit...
``

File: boilerplate/flyte/docker_build/Readme.rst [hunk 6]
```diff
@@ -11,13 +11,13 @@ If git head has a git tag, the Dockerhub image will also be tagged ``<IMAGE>:<GI
 
 **To Enable:**
 
-Add ``lyft/docker_build`` to your ``boilerplate/update.cfg`` file.
+Add ``flyteorg/docker_build`` to your ``boilerplate/update.cfg`` file.
 
-Add ``include boilerplate/lyft/docker_build/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
+Add ``include boilerplate/flyte/docker_build/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
```
Comment (EngHabu) [inline boilerplate/flyte/docker_build/Readme.rst:16]: should this be flyteorg?
```suggestion
Add ``include boilerplate/flyteorg/docker_build/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
```

File: boilerplate/flyte/golang_test_targets/Readme.rst [hunk 7]
```diff
@@ -15,17 +15,17 @@ Provides a ``test_benchmark`` target for benchmark tests.
...
 
-Add ``lyft/golang_test_targets`` to your ``boilerplate/update.cfg`` file.
+Add ``flyteorg/golang_test_targets`` to your ``boilerplate/update.cfg`` file.
 
 Make sure you're using ``go mod`` for dependency management.
 
 Provide a ``.golangci`` configuration (the lint target requires it).
 
-Add ``include boilerplate/lyft/golang_test_targets/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
+Add ``include boilerplate/flyte/golang_test_targets/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
```
Comment (EngHabu) [inline boilerplate/flyte/golang_test_targets/Readme.rst:24]: I'm a bit confused about flyte or flyteorg here

File: boilerplate/flyte/docker_build/Readme.rst (hunk 6 above)
Comment (evalsocket) [reply boilerplate/flyte/docker_build/Readme.rst:16]: yes, but I just use the same names that I found in the boilerplate.

Comment (wild-endeavor) [reply boilerplate/flyte/docker_build/Readme.rst:16]: it's just `flyte` for now - my vote is to leave this as is for now, can change in the future.

Comment (wild-endeavor) [reply boilerplate/flyte/docker_build/Readme.rst:16]: https://github.com/flyteorg/boilerplate/tree/master/boilerplate/flyte

File: boilerplate/flyte/end2end/end2end.sh [hunk 8]
```diff
@@ -31,4 +31,4 @@ fi
 
 make kustomize
 make end2end_execute
-popd
+popd
```
Comment (wild-endeavor) [inline boilerplate/flyte/end2end/end2end.sh:34]: can we add back newlines at some point?  i really like newlines.

Comment (wild-endeavor) [reply boilerplate/flyte/end2end/end2end.sh:34]: doesn't have to be this PR.

File: pull_request_template.md (hunk 3 above)
Comment (wild-endeavor) [reply pull_request_template.md:1]: @evalsocket could you clarify - is this no longer true?

**Review by EngHabu — APPROVED** [review]

File: boilerplate/flyte/end2end/Makefile (hunk 5 above)
Comment (EngHabu) [reply boilerplate/flyte/end2end/Makefile:8]: Do you still need this?

**Review by EngHabu — CHANGES_REQUESTED** [review]

File: boilerplate/flyte/end2end/Makefile (hunk 5 above)
Comment (evalsocket) [reply boilerplate/flyte/end2end/Makefile:8]: No

File: boilerplate/flyte/end2end/end2end.sh (hunk 8 above)
Comment (evalsocket) [reply boilerplate/flyte/end2end/end2end.sh:34]: Yes i will add in next iteration of boilerplate

**Comment by evalsocket** [comment]
We don't need chmod but we can leave it for now and create a separate PR in boilerplate and boilerplate automation will create a pr in flyteadmin.

File: boilerplate/flyte/pull_request_template/Readme.rst [hunk 9]
```diff
@@ -5,4 +5,4 @@ Provides a Pull Request template.
 
 **To Enable:**
 
-Add ``lyft/golang_test_targets`` to your ``boilerplate/update.cfg`` file.
+Add ``flyteorg/golang_test_targets`` to your ``boilerplate/update.cfg`` file.
```
Comment (wild-endeavor) [inline boilerplate/flyte/pull_request_template/Readme.rst:8]:
```suggestion
Add ``flyte/golang_test_targets`` to your ``boilerplate/update.cfg`` file.
```

Comment (wild-endeavor) [reply boilerplate/flyte/pull_request_template/Readme.rst:8]: right?

**Comment by wild-endeavor** [comment]
Also had a question - is the `#patch`, `#minor`, `#major` stuff no longer true?  How is that controlled?  I noticed it disappeared from the pr template.

File: boilerplate/flyte/end2end/Makefile (hunk 5 above)
Comment (wild-endeavor) [inline boilerplate/flyte/end2end/Makefile:8]: Is this still needed?  I thought not?

File: pull_request_template.md (hunk 3 above)
Comment (wild-endeavor) [reply pull_request_template.md:1]: ping

File: boilerplate/flyte/end2end/Makefile (hunk 5 above)
Comment (evalsocket) [reply boilerplate/flyte/end2end/Makefile:8]: we can leave it for now and boilerplate automation will automatically create a PR. Fix is needed in boilerplate

File: pull_request_template.md (hunk 3 above)
Comment (evalsocket) [reply pull_request_template.md:1]: it is coming from boilerplate, I didn't make any changes in pull_request template

**Review by katrogan — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Pull request merged**